plugins/generator-1.20.1/forge-1.20.1/aitasks/break_blocks.java.ftl

```diff
@@ -1,4 +1,4 @@
-<#include "mcitems.ftl">
 <#include "aiconditions.java.ftl">
+<#include "mcitems.ftl">

```

Comment:
Please keep this file intact as the previous structuring with a newline after include was preferred (we try to add newline after FTL includes).

Reordering of includes also does not have to do much with porting and is in this view unnecessary change for this PR, applies to other AI tasks too